core/scripts/cli/sponsor/redeem.js

```diff
@@ -1,5 +1,49 @@
+const inquirer = require("inquirer");
+const getDefaultAccount = require("../wallet/getDefaultAccount");
+
 const redeem = async (web3, artifacts, emp) => {
-  console.log("In redeem flow");
+  const { fromWei, toWei, toBN } = web3.utils;
+  const SyntheticToken = artifacts.require("SyntheticToken");
+  const sponsorAddress = await getDefaultAccount(web3);
+  const collateral = (await emp.getCollateral(sponsorAddress)).toString();
+  const position = await emp.positions(sponsorAddress);
+
+  const tokenAddress = await emp.tokenCurrency();
+  const token = await SyntheticToken.at(tokenAddress);
+  const walletTokens = await token.balanceOf(sponsorAddress);
+
+  console.log("You have:");
+  console.log(
+    "Position: " +
+      fromWei(collateral) +
+      " WETH backing " +
```

Comment:
Just curious: do we do a check anywhere in the CLI to make sure that  the collateral currency being used is WETH? As previously discussed, we are forcing the collateral currency to be WETH so just want to make sure that the CLI doesn't "work" with non-weth-backed synthetic tokens.

Reply:
No, we don't currently. Although based on the recent Slack thread, maybe we're supporting more than WETH, so we might have to reexamine this whole assumption.

I'm trying to come up with a good way to do that, something that all the methods can reuse. Any thoughts around that?